bentoml/_internal/bento/docker/entrypoint.sh

```diff
@@ -11,8 +11,16 @@ _is_sourced() {
 
 _main() {
 	# if first arg looks like a flag
-	if [ "${1:0:1}" = '-' ]; then
-		set -- bentoml serve --production "$@" "$BENTO_PATH"
+	if [ -z "$1" ] || [ "${1:0:1}" = '-' ]; then
```

Comment:
could we do this at the CLI level instead?

e.g.
```
bentoml serve BENTO --runner=MY_RUNNER
```

Reply:
It could be another PR? This PR is mainly for yatai deployment.

Reply:
@bojiang I think we should just do that now for Yatai deployment, this change still requires Yatai to set an env var right? we can achieve the same via CLI arg, but a lot cleaner